packages/dashboard/src/components/iot-dashboard/iot-dashboard-widget/iot-dashboard-widget.css

```diff
@@ -0,0 +1,27 @@
+.widget {
+  background-color: white;
+  font-weight: bold;
+  height: 100%;
+  user-select: none;
+  box-sizing: border-box;
+}
+
+.widget:hover {cursor: all-scroll;
```

Comment:
Maybe we could add style lint to require new lines for blocks? https://stylelint.io/user-guide/rules/list#block-1

Reply:
fixed in revision